### ISSUE TITLE
ObjC nullability warning

### ISSUE THREAD

**Issue by borut-t** [onfido/onfido-ios-sdk#79]
### What was the expected behaviour?

Not see warning in Xcode

### What happened instead?

Warning in Xcode

### Version info:

- Cocoapods (if applicable): 1.6.1
- Onfido SDK (Debug/Release): 11.0.0
- iOS: 12.2
- Xcode: 10.2
- Device/Simulator: both
- Device/Simulator language: en

### Integration configuration:

standard

### Steps to reproduce:

Build the project with the latest Xcode 10.2
```
Pods/Onfido/Onfido.framework/Headers/OnfidoSEGGzipData.h:12:48: Pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)
```

<img width="1139" alt="Screenshot 2019-04-01 at 15 27 56" src="https://user-images.githubusercontent.com/677166/55331047-c5ae3e80-5492-11e9-8d79-3858f6f45a21.png">

**Comment by anuragajwani** [comment]
@borut-t I just created a new project from scratch and I am unable to see this warning. Is there any project configuration that raises this warning?

Anurag

**Comment by borut-t** [comment]
Nothing special. I've just upgraded from `10.6.0` to `11.0.0` and build on Xcode 10.2.
There was no warning on `10.6.0` using Xcode 10.1

**Comment by compyman** [comment]
I am also seeing this (although for some reason it is an Error for me, and was preventing the build (maybe since we are also using swift?))
On Xcode 10.2 with sdk 11.0.0 I edited the header file to include a `_Nullable` specifier and it fixed my build

**Comment by anuragajwani** [comment]
thanks for your reply @compyman, I am still unable to get a warning/error from this class from a new project created from scratch. There should be a Xcode project configuration to turn these on and display either warning or error. I will try to look for this option.

Thanks for posting your fix,
Anurag

**Comment by compyman** [comment]
Hi @anuragajwani how is your new project set up & which version of xcode are you on?
I'm happy to help debug if you want (I also had a ticket open with your support team)

**Comment by anuragajwani** [comment]
Hi @compyman,

Thanks for helping out, I really appreciate it. I created new project from scratch using Xcode 10.2, ran `pod init` and added `pod 'Onfido', '11.0.0'` to the `Podfile`. Next ran `pod install` and used the SampleApps Onfido configuration. Added my token and then ran it on simulator.

I didn't touch the project configuration. I believe its in the project configuration I'd be able to throw these warnings/errors hence my first response.

Anurag

**Comment by borut-t** [comment]
@anuragajwani just look at the code in the SDK and you'll notice what's missing in the code.

**Comment by anuragajwani** [comment]
@borut-t I am not sure if I follow you last comment. The warning is coming from SegmentSDK which is bundled inside of our own SDK. 

There is nothing missing in code (not sure what you meant by that). Not all of our customers are experiencing this issue only a subset of customer which leads to my thinking that the issue could potentially lie in project configuration.

Anurag

**Comment by borut-t** [comment]
Correct, this is in your SDK. And from what I can read, the code is missing nullability specifier.

**Comment by anuragajwani** [comment]
@borut-t I'll add the changes in the next release however I am unable to reproduce hence unable to confirm before the release that I have in fact removed the warning/error. If you can point me to your Xcode project configuration that triggers this warning I will trigger it on my side too. This is what I would like to figure out.

**Comment by compyman** [comment]
are you using the SampleApps or the SampleAppsObjectiveC project? 
Try using the objective c - or just importing the sdk into a new project then  importing and initalizing something? 
I was able to fix it by 'unlocking' the header for editing and adding the `_Nullable` attribute. I don't think there is a project configuration for making this an error because we certainly wouldn't have it enabled (it's very annoying)

**Comment by anuragajwani** [comment]
@compyman I created a new Swift project. I didn't use the SampleApp as I wanted to test it out without any state/configuration changes. Thanks for the pointer I'll try to reproduce it on an Objective-C project. It's a bit late in the day in London now so will try to test it out tomorrow.

**Comment by anuragajwani** [comment]
@compyman I was able to reproduce it on the SampleAppObjC project. Thanks for helping out 🙂.

We will include the fix on the next release.
Anurag

**Comment by compyman** [comment]
@anuragajwani great - do you have an idea when that will be up?

**Comment by anuragajwani** [comment]
@compyman we are aiming for a patch release on Monday

**Comment by anuragajwani** [comment]
we have been delayed a little bit today. We are trying to release the fix asap.

**Comment by anuragajwani** [comment]
@compyman @borut-t we have released iOS SDK 11.0.1 with a fix for this problem. Please let me know if it has solved your issue.

Sorry for the inconvenience caused,
Anurag

**Comment by borut-t** [comment]
@anuragajwani it looks like this fix works. I don't see warning anymore. Thanks 👍
